server/game/cards/06.6-TBWB/Patchface.js

```diff
@@ -16,17 +15,8 @@ class Patchface extends DrawCard {
     }
 
     getFoolKeywords() {
-        let foolKeywords = [];
         let fools = this.game.filterCardsInPlay(card => card.getType() === 'character' && card.hasTrait('Fool') && card !== this);
-
-        for(let card of fools) {
-            for(let keyword of Keywords) {
-                if(card.hasKeyword(keyword)) {
-                    foolKeywords.push(keyword);
-                }
-            }
-        }
-
+        let foolKeywords = [...new Set(flatten(fools.map(card => card.getKeywords())))];
```

Comment:
Is this to ensure Patchface can gain Bestow? Does this actually work as he enters play?

Reply:
the patchface change enables him to gain the correct versions of "No attachment except X" keywords, bestow wasn´t on my radar, but it should also work in the sense that he gains it (but no bestwo trigger is prompted as he doesn´t enter play).